views/index.njk

```diff
@@ -139,11 +137,12 @@
             query.prohibited = [];
 
             $('#collapse-moderation').find(":checked").each(function() {
-                if($(this).val() == "moderated"){
+                if ($(this).val() == "moderated"){
```

Comment:
Please don't change code formatting. Applies to spaces and brackets added in the whole PR.